src/apps/service/apps/de/listings.clj

```diff
@@ -134,13 +140,32 @@
   [_ workspace params]
   (list-shared-apps workspace (workspace-favorites-app-category-index) params))
 
+(defn format-cyverse-blessed-category
+  "Formats the virtual group for apps that are certified as blessed"
+  [_ workspace params]
+   {:system_id de-system-id
+    :id        cyverse-blessed-id
+    :name      "CyVerse Blessed"
+    :is_public false
+    :total     (count-apps-for-user nil (:id workspace) {:app-ids (:listing-ids params)})})
+
+(defn list-cyverse-blessed-apps
+  [user workspace params]
+   (let [blessed-ids (app-ids->certified-ids-set (:username user) (:app-ids params))]
```

Comment:
This seems to work because `list-apps-in-group` already makes a call out to permissions `load-app-permissions` which returns the list of all apps the user has permission to view.  Then it calls `augment-listing-params` to add that list as `:app-ids` to `params`.  So here, I'm just taking that list of `:app-ds` and filtering it down to the blessed ids.